utils/squid.py

```
@@ -0,0 +1,60 @@
+        encoding='utf-8'
+    )
+    return result.stdout
+
+
+def start_squid_container(container_name='squid-proxy'):
+    """Start squid proxy container"""
+    project_root = Path(os.getenv(constants.PROJECT_PATH))
+    config_path = project_root / 'squid' / 'squid.conf'
+
+    if not config_path.exists():
+        raise FileNotFoundError(f"Config file not found at {config_path}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)